geomesa-tools-common/src/main/scala/org/locationtech/geomesa/tools/common/CommonParams.scala

```diff
@@ -64,12 +64,12 @@ trait OptionalDTGParam {
 
 trait OptionalAttributesParam {
   @Parameter(names = Array("-a", "--attributes"), description = "Attributes to evaluate (comma-separated)")
-  var attributes: java.util.List[String] = null
+  var attributes: String = null
```

Comment:
I've converted these all back to Lists but I had to touch a lot of places to get the typing right.